lib/themes/dosomething/paraneue_dosomething/js/utilities/Globe.js

```diff
@@ -0,0 +1,12 @@
+import $ from 'jquery';
+import setting from './Setting';
+
+function init() {
+  const globeUpdateUrl = setting('dosomethingSetting.globeUrl');
+  if (globeUpdateUrl !== undefined) {
```

Comment:
I can add an empty string check for saftey but I actually need an undefined check because the preprocess logic only runs when the dosomething_setting variable is set. 

```
  $update_url = variable_get('dosomething_settings_realtimefeed_update_url');
  if (!empty($update_url)) {
    drupal_add_js(
```

So if I don't set the variable, refresh the page and prinout the globalUpdateUrl, it's actually undefined

![screen shot 2015-11-23 at 9 38 03 am](https://cloud.githubusercontent.com/assets/897368/11339155/25371c20-91c6-11e5-83ab-e2c67634e556.png)